components/Footer.tsx

```diff
@@ -65,7 +65,7 @@ export const Footer: FC = () => {
 
           <FooterItem title="Accessoires">
             <div className="accessoires-link">
-              <ul className="accessoires-link__first">
+              <ul className="accessoires-link__list">
```

Comment:
`<ul className="accessoires-links">` instead `<ul className="accessoires-link__list">`

links means a list of links :)